lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -120,7 +120,18 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       const value = audit.result.numericValue ? audit.result.numericValue.toString() : 'null';
       return [audit.id, value];
     });
-    const params = new URLSearchParams(metricPairs);
+    const paramPairs = [...metricPairs];
+
+    if (this.detailsRenderer.report && this.detailsRenderer.report.configSettings.emulatedFormFactor !== 'none') {
```

Comment:
```suggestion
    if (this.detailsRenderer.report) {
```

do we even need to check the second half? seems fine to bring 'none' over.

Reply:
what scoring is even used for `none`, mobile or desktop? I assume desktop.

either way the score calc will have to have explicit knowledge of what to do when `device === 'none' or device === undefined` sooooo sure let's just pass it thru.